db/src/main/resources/db/migration/V1_042__add_card_status.sql

```diff
@@ -0,0 +1,4 @@
+alter table claim add column card_status varchar(50);
+update claim set card_status = 'ACTIVE' where claim.claim_status = 'ACTIVE' or claim.claim_status = 'NEW';
```

Comment:
As discussed we can't keep new claims as null because they'll stay null if they move to active before the code that updates the card status goes in.
Updated to set card status timestamp.